Model/Exceptions/AlmaProductException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Alma\MonthlyPayments\Model\Exceptions;
+
+use Exception;
+
+class AlmaProductException extends Exception
+{
```

Comment:
you can extends the construct to  set the exception message mandatory or put a default message

Reply:
good idea